src/operationpoint/power_flow.jl

```diff
@@ -52,6 +52,15 @@ function make_generator!(dict::Dict{String,Any}, key_b::Int, node::Union{VSIMini
     ((dict["gen"])[string(key)])["qmax"] = 1.5 * abs(node.Q)
 end
 
+function make_generator!(dict::Dict{String,Any}, key_b::Int, node::NormalForm)
+    make_generator_header(dict, key_b)
+    key = length(dict["gen"])
+    ((dict["gen"])[string(key)])["pg"] = node.P
+    ((dict["gen"])[string(key)])["pmin"] = 0.9 * node.P
+    ((dict["gen"])[string(key)])["pmax"] = 1.1 * node.P
```

Comment:
I agree but this is an issue that should be dealt with for all models that can be used with `solve_powerflow`. I used the numbers for `pmin` and `pmax` because they were used in the other models and I could not find any other documentation regarding them. Especially I would like to understand the choice of values for the reactive power in the other models.

Reply:
I opened a new issue #204 for this.